lib/layer.js

```diff
@@ -48,6 +48,33 @@ function Layer(path, options, fn) {
 }
 
 /**
+ * Handle any request for the layer.
+ *
+ * @api private
+ */
+
+Layer.prototype._handle_any = function _handle_any(/* ...args, next */) {
+  var self = this
+  var args = Array.prototype.slice.call(arguments)
+  var next = args.pop()
```

Comment:
Don't `pop` and `concat` later, you could just use `arguments[arguments.length - 1]` and apply using `arguments`.